tests/unittest_brain_numpy_core_einsumfunc.py

```diff
@@ -0,0 +1,80 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+from __future__ import annotations
+
+import unittest
+
+import pytest
+
+try:
+    import numpy  # pylint: disable=unused-import
+
+    HAS_NUMPY = True
+except ImportError:
+    HAS_NUMPY = False
+
+from astroid import builder
+
+
+@unittest.skipUnless(HAS_NUMPY, "This test requires the numpy library.")
+class BrainNumpyCoreFromNumericTest(unittest.TestCase):
+    """
+    Test the numpy core einsumfunc brain module
+    """
```

Comment:
```suggestion
    """Test the numpy core einsumfunc brain module."""
```

At some point we should turn on `pydocstringformatter` here as well.